### DISCUSSION TITLE
[Feature] sort albums by picture title

### DISCUSSION THREAD

**Discussion by reitenth** [immich-app/immich#2662] [Feature Request]
### The feature

its kind of impossible to find specific pictures right now as you can not sort by title. thanks.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by logicalpeyote777** [comment]
If you are using cloudflare free, it have a limit of 100M, would be nice if the client can split the upload in 99M chunks for large files

**Comment by bo0tzz** [comment]
#1689
